src/index.html

```diff
@@ -214,49 +214,70 @@
-                            </div>
+                            Submissions to the EVA consist of VCF file(s), any associated data file(s), and metadata that describe sample(s), experiment(s), and analysis that produced the variant and/or genotype call(s).
+                            This metadata is described in an Excel template that can be found <a href="files/EVA_Submission_template.V1.0.4.xlsx">here</a>. Please also see <a href="files/EVA_Submission_template.V1.0.4_mockup.xlsx">here</a> for a mocked up version of this template that has been completed for a fictional study.
+                        </p>
+                        <p>
+                            VCF file(s) submitted to the EVA must be truly valid a 4.X version of the file format <a href="http://samtools.github.io/hts-specs/VCFv4.3.pdf" target="_blank">specification</a>.
+                            Files can be validated prior to submission using our validation suite that is available via our <a href="https://github.com/EBIvariation/vcf-validator" target="_blank">EVA VCF validation suite</a>.
                         </p>
-                        <p>To submit data to EVA please download this  <a href="files/SubmissionTemplate.xlsx" target="_blank"  onClick="ga('send', 'event', { eventCategory: 'Download', eventAction: 'Click', eventLabel: 'SubmissionTemplate.xlsx'});">template</a>, fill in and send by email to <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a>.</p>
                     </div>
 
                     <h4><span class="icon icon-functional" data-icon="_"></span> Submit</h4>
                     <div>
-                        <p>Upload your data files to your private submission upload account or directly to the <a href="mailto:eva-helpdesk@ebi.ac.uk?subject=Sequence submissionto EVA&amp;body=**We will provide you with a submission package metadata templates guides for completing templates**" target="_top">eva-helpdesk@ebi.ac.uk</a>.<br/>
-                            To improve submission processing time, users are encouraged to prevalidate submitted files using the <a href="https://github.com/EBIvariation/vcf-validator" target="_blank">EVA VCF validation suite</a>; more  information can be found at our <a href="?FAQ#collapseThree">FAQs</a>.</p>
+                        <p>Upload your VCF file(s), associated data file(s) and EVA metadata template to your private EVA FTP location.</p>
                     </div>
 
-                    <h3>Converting genetic variation data to VCF</h3>
+                    <h4><span class="icon icon-functional" data-icon="k" aria-hidden="true"></span> Receive</h4>
+                    <div>
+                      <p>The EVA aims to process submissions within two business days. Accession numbers shall be sent via email to the submitter upon successful archival of the deposited data.</p>
+                    </div>
+
+                    <h3>EVA data requirements</h3>

```

Comment:
@garysaunders Shouldn't the requirements be described before the submission stages? Otherwise people could prepare submissions that don't satisfy requirements like the INSDC accessioned assembly, or not notice the remark about clinical submissions.